src/AppBundle/Resources/views/admin/tags.html.twig

```diff
@@ -22,6 +22,12 @@
   <td><a href="{{ path('admin_tag', { slug: tag.slug }) }}">#{{ tag.id }}</a></td>
   <td>{{ tag.name }}</td>
   <td><span class="label label-default" style="background-color: {{ tag.color }};">{{ tag.color }}</span></td>
+  <td>
```

Comment:
- You should also add a (empty) `<th>` in the `<thead>` section so that number of columns match
- Would look better with the button right-aligned (use Bootstrap's `text-right` class)